openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -44,6 +46,28 @@ export class CourseItemsListingView {
       },
     });
 
+    const esgEnabled = context.ENHANCED_STAFF_GRADER;
+    const esgRootUrl = context.ORA_GRADING_MICROFRONTEND_URL;
+
+    const ESGCell = Backgrid.UriCell.extend({
+      render() {
+        this.$el.empty();
+        const name = this.column.get('name');
+        this.$el.addClass(name);
+        const displayValue = itemViewEnabled && esgEnabled ? gettext('View and grade responses') : gettext('Demo the new Grading Experience');
```

Comment:
I think this is wrong. I don't think we should care about itemViewEnabled here.

if (!itemViewEnabled, esgEnabled), this would show "Demo the new Grading Experience" which is not correct.

Reply:
itemViewEnabled is a feature where we render content from the Xblock into the page, which is what we're doing in the cases where we were doing the in-page staff and waiting step stuff. This is just an external link